docs/content/v2024.2/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md

```diff
@@ -46,12 +46,8 @@ To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super A
 
 ## Use OIDC groups with YugabyteDB Anywhere roles
 
-This feature is {{<tags/feature/ea>}}.
-
 If your OIDC provider is configured with user groups, you can map the groups to [YugabyteDB Anywhere roles](../anywhere-rbac/). Users who are members of these groups can then sign in to YugabyteDB Anywhere without needing to be added to YugabyteDB Anywhere first. Users who are members of multiple groups are assigned the most privileged role.
 
-Currently, groups can only be mapped to [built-in](../anywhere-rbac/#built-in-roles) roles.
-
 Note that, if you use group mapping, you must manage users via your OIDC server. You can't add or change user roles in YugabyteDB Anywhere.
 
```

Comment:
Also worth mentioning here that for every user (except SuperAdmin), the role determined via group mapping will override any role assigned previously.